test/integration/exceptionalOpeningTimes.js

```diff
@@ -18,38 +18,55 @@ function mockCurrentDate2() {
 }
 
```

Comment:
The mock current dates functions above this are calling new Date() with string parameters, when they should be integers.

To people not familiar with the peculiarities of the js date library it looks like new Date(2017, 4, 25) would create the date  created is 25/4/2017, when actually it is creating 25th of May 2017. It might make it a little clearer if a comment was added or the month integer was extracted at as a const called 'May'